File: online-judge-solutions/Codeforces/1303A.c
//https://codeforces.com/problemset/problem/1303/A
//Lang		:	GNU C11
//Time		:	15 ms
//Memory	:	0 KB
#include <stdio.h>
#include <string.h>
int main(){
	int t;
	scanf("%d",&t);
	while(t--){
		char s[100],*p;
		scanf("%s",s);
		p=strchr(s,'1');
		int l=p-s;
		p=strrchr(s,'1');
		int r=p-s;
		int c=0;
		for(int i=0;i<strlen(s);i++){
			if(s[i]=='1') c++;
		}
		c==0 ? printf("0\n"):printf("%d\n",(r-l+1)-c);
	}
	return 0;
}

File: online-judge-solutions/Codeforces/1303A.py
#https://codeforces.com/problemset/problem/1303/A
#Lang		:	Python 3.8
#Time		:	31 ms
#Memory     :	0 KB
t=int(input())
for x in range(0,t):
    s=input()
    l=s.find('1')
    r=s.rfind('1')
    c=s.count('1')
    print(0) if c==0 else print((r-l+1)-c)
